src/cloud/docker/docker-quick-reference.md

```diff
@@ -49,13 +49,13 @@ See [Service versions] for a list of the options to configure the software servi
 
 ## bin/magento-docker
 
-Run `bin/magento-docker` commands using the following format:
+Use `bin/magento-docker` commands to simplify running docker-compose tasks. For example, instead of running separate docker-compose build, deploy, and post-deploy commands to redeploy Magento in a Docker environment, you can run the following command:
 
 ```bash
-./bin/magento-docker <command>
+./bin/magento-docker ece-redeploy
 ```
 
-For example, to connect to the bash shell:
+The following example shows the output for the `./bin/magento-docker` command to connect to the bash shell:
```

Comment:
No need to repeat the command and add extra fluff. Already, people find this stuff wordy.

Reply:
I added the sentence because it doesn't flow from the previous paragraph. I don't think this example is even necessary.